src/Microsoft.Kusto.ServiceLayer/DataSource/Contracts/KustoConnectionDetails.cs

```diff
@@ -0,0 +1,11 @@
+namespace Microsoft.Kusto.ServiceLayer.DataSource.Contracts
```

Comment:
We typically have used the `Contacts` namespace to define types that are passed over the JSON-RPC communication interface.  And define internal "object model" classes somewhere else.  Not sure if this is intended for IPC messaging.